logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/mlsmigration/MLSMigrator.kt

```diff
@@ -83,13 +94,32 @@ internal class MLSMigratorImpl(
 
     private suspend fun finalise(conversationId: ConversationId): Either<CoreFailure, Unit> {
         kaliumLogger.i("finalising ${conversationId.toLogString()} to mls")
-        return conversationRepository.updateProtocol(conversationId, Conversation.Protocol.MLS)
-            .flatMapLeft {
-                kaliumLogger.w("failed to migrate ${conversationId.toLogString()} to mls: $it")
+        return conversationRepository.updateProtocol(conversationId, Protocol.MLS)
+            .fold({ failure ->
+                kaliumLogger.w("failed to migrate ${conversationId.toLogString()} to mls: $failure")
```

Comment:
I suggest to have the same terminology here, since we have this code in the update protocol, let's use the keyword "update conversation protocol to..."; I think that would be easy to follow later.

Reply:
also here we don't really migrate, we finalising and updating the protocol.

Reply:
Updated